packages/@react-aria/radio/src/useRadio.ts

```diff
@@ -86,7 +86,7 @@ export function useRadio(props: RadioAriaProps, state: RadioGroupState, ref: Ref
       name: radioGroupNames.get(state),
       tabIndex,
       disabled: isDisabled,
-      readOnly: isReadOnly,
+      'aria-readonly': isReadOnly,
```

Comment:
We may want to do something like this since false is the default:

```suggestion
      'aria-readonly': isReadOnly || undefined,
```

Reply:
Yes! Fixed